### PR TITLE
Fix picked locale on swedish quotes also

### PR DESCRIPTION
**Pull request by palmenhq**
Bonus: Remove the debugger (it's just confusing now that we have the proper flow).

A lot of code removal here: but this is the juicy stuff: https://github.com/HedvigInsurance/web-onboarding/pull/283/files#diff-d61a9468bd6799cdb7b19b106b2d51cfR56-R60

### CODE REVIEW

**Review by meletios — APPROVED** [review]
🇰🇵

File: src/pages/Embark/createQuote.ts [hunk 1]
```diff
@@ -53,24 +52,12 @@ const MUTATION = gql`
...
 
-  if (!storage.session.getSession().token) {
-    const sessionResult = await apolloClient.client.mutate({
-      mutation: CREATE_SESSION_TOKEN_MUTATION,
-    })
-    await afterTick(() => {
-      apolloClient!.subscriptionClient.close(true, true)
-      storage.setToken(sessionResult.data.createSessionV2.token)
-    })
-  }
```
Comment (vonElfvin) [inline src/pages/Embark/createQuote.ts]: I assume you wanted to remove this?

**Review by vonElfvin — APPROVED** [review]
🇫🇰

File: src/pages/Embark/createQuote.ts (hunk 1 above)
Comment (palmenhq) [reply src/pages/Embark/createQuote.ts]: Yes. This pr is an amendment to #279, and this now happens inside  [`setupSession`](https://github.com/HedvigInsurance/web-onboarding/pull/279/files#diff-d8aadba1570390110355bbcd32043b06R40-R72) - plus the actual market picker which should happen there also :) unfortunately someone has done quite some copy-pasting of this logic and i missed one place.

**Pull request merged**